src/main/java/com/iot/lab/manager/UserManager.java

```diff
@@ -0,0 +1,38 @@
+package com.iot.lab.manager;
+
+import com.iot.lab.enums.Services;
+import com.iot.lab.modals.User;
+import lombok.AllArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+public class UserManager {
+
+    private final List<User> objects;
+
+    public List<User> sortByFullName(final boolean sortOrder) {
```

Comment:
sortOrder is not a good name fo a boolean property - I have no idea what it means